appendices/F/1_flip_int/palindrome/palindrome_test.go

```diff
@@ -0,0 +1,31 @@
+package palindrome
+
+import (
+	"slices"
+	"testing"
+)
+
+func FuzzIsPalindromeNumber(f *testing.F) {
+	// Seed corpus with some basic test cases
+	f.Add("1221") // nominal case
+	f.Add("")     // empty string
```

Comment:
We add this test case here as an edge case that checks that our code is "soild" enough.